### PR TITLE
Fix changelog text

### LINKED ISSUE #305: Misleading title for breaking-changes and changelog
**Describe the bug**
The title for breaking-changes and changelog reads "Backward compatibility errors (<some number)".
This is misleading because it refers to all findings as errors which some may be warn-level or info-level.

**To Reproduce**
1. Run `oasdiff changelog data/openapi-test1.yaml data/openapi-test3.yaml`

**Expected behavior**
Title should specify the number of errs, warns and infos.

### PR DESCRIPTION
**Pull request by blva**
Fixes https://github.com/Tufin/oasdiff/issues/305

Example:

```shell

Total API changes (1):
    [info (0)| error (1)| warning (0)]
error   [request-parameter-default-value-changed] at ChangeToDefaultHeaderParameterValues-revision.yaml
        in API GET /api/changeToDefaultHeaderParameterValues
                for the 'header' request parameter 'headerParam', default value was changed from 'default' to 'newDefault'
```
Screenshot 2023-07-04 at 15 41 39

### CODE REVIEW

File: checker/localizations_src/en/messages.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ in: in
 added-required-request-body: added required request body
 request-parameter-removed: deleted the %s request parameter %s
 total-errors: "Backward compatibility errors (%d):\\n"
+total-changes: "Total API changes (%d):\\n    [info (%d)| error (%d)| warning (%d)]\\n"
```
Comment (blva) [inline checker/localizations_src/en/messages.yaml]: cc @reuvenharrison @tibulca feel free to propose anything else

File: internal/changelog.go [hunk 2]
```diff
@@ -137,7 +137,10 @@ func outputChangelog(config checker.BackwardCompatibilityCheckConfig, format str
 		}
 	case FormatText:
 		if len(errs) > 0 {
-			fmt.Fprintf(stdout, config.Localizer.Get("messages.total-errors"), len(errs))
+			infoCount := getLevelCount(errs, checker.INFO)
+			warnCount := getLevelCount(errs, checker.WARN)
+			errCount := getLevelCount(errs, checker.ERR)
+			fmt.Fprintf(stdout, config.Localizer.Get("messages.total-changes"), len(errs), infoCount, errCount, warnCount)
```
Comment (reuvenharrison) [inline internal/changelog.go]: It would be nice to display the color-codes for 'info', 'err' and 'warning'

Comment (blva) [reply internal/changelog.go]: Thanks for the suggestion, added.

**Review by reuvenharrison — APPROVED** [review]
I made some changes to the title format and I'm going to merge this PR.
There is still one problem with piped output which I will open a separate issue to track.
Thank you!

**Pull request merged**